instrumentation/jdbi3/pom.xml

```
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
+<!--
+
+    Copyright The OpenZipkin Authors
+    SPDX-License-Identifier: Apache-2.0
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>io.zipkin.brave</groupId>
+    <artifactId>brave-instrumentation-parent</artifactId>
+    <version>6.2.1-SNAPSHOT</version>
```

6.2.0-SNAPSHOT, but should be universal across all files even the bom

```suggestion
    <version>6.2.0-SNAPSHOT</version>
```

---

No this was my mistake. Whatever the next minor is would be the right version as we don't typically add new instrumentation in a patch

---

Bumped to 6.3.0-SNAPSHOT everywhere